EIPS/eip-tbd.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Standardizing Smart Account Feature Discovery for apps
+description: Enable Applications to Seize Smart Account features such as Sponsored Transactions and Batched Transactions
+author: Derek Rein (@arein), Kristof Gazso (@kristofgazso)
+discussions-to: <URL> - TBD
+status: Draft
+type: Standards Track
+category: ERC
+created: 2024-11-24
+requires: ERC-4337, CAIP-25
+
+## Abstract
+
+Enable wallets to advertise support of ERC-4337 and their batched transactions Application Binary Interface (ABI) in a standardized way such that end-user applications can leverage these features.
+
+## Motivation
+
+Smart Accounts are on the rise and ERC-4337 has established itself as the standard that is implemented by the vast majority of them, allowing them to plug into a widespread network of bundlers and paymasters. Most Smart Accounts implementations also support batched transactions albeit via different interfaces.
+
+Discovering an end-user's wallet's support for these features is very useful to application developers who want to build more seamless applications for their end-users that require fewer clicks.
+
+### Case Study 1: Ethereum Name Service (ENS)
+
+Take ENS, a popular Naming Service, for example. Their current registration experience requires multiple transactions (i.e. clicks) from the end-user. Managing the blockchain state of these transactions in the user-interface is not easy and a scary experience for end-users who are not confident that the process is idempotent and they can refresh the browser.
+
+Batched transactions can transform this into a single click experience.
+
+### Case Study 2: peanut.to
+
+Peanut.to is a popular payment link service allowing end-users to send tokens to other users without knowing them or asking them their preferred address in advance of sending the funds.
+
+Peanut might want to sponsor the end-user's gas fee and subtract that instead from the funds send, as well as batching the currently two transactions into a single click.
+
+## Specification
+
+We will build this on top of [CAIP-25](https://github.com/ChainAgnostic/CAIPs/blob/main/CAIPs/caip-25.md) which most wallets already implement.
+
+Here is an example reponse:
```

Comment:
what is the request?

Reply:
You can see it in the standard: https://github.com/ChainAgnostic/CAIPs/blob/main/CAIPs/caip-25.md